src/Bindings/PluginManager.cpp

```diff
@@ -1097,6 +1097,18 @@ bool cPluginManager::CallHookProjectileHitEntity(cProjectileEntity & a_Projectil
 
 
 
+bool cPluginManager::CallHookResourcePack(const UInt8 a_Status)
+{
+	return GenericCallHook(HOOK_RESOURCEPACK, [&](cPlugin * a_Plugin){
```

Comment:
small syntax error here, I made a pr on your fork for that. That's the reason it doesn't pass jenkins 
https://github.com/plan1231/cuberite/pull/2

Reply:
I merged that PR but I think it got merged into another branch, so it's not showing up here. Would it be possible to create a PR that merges into `plan1231:resource-pack`?

Reply:
Well, actually there is only one character diff, maybe you can sort it out by yourself ? I don't have access to a PC till tomorrow and it seems like I can't do pr on my phone 😅

Reply:
Oh shoot sorry! I didn't know earlier, because I didn't know how to get the diff between your new commit and this branch, because the PR's target was the master branch. I'll definitely go learn git better so that I don't take up your time with requests like this in the future

Reply:
Don't worry, I can see the confusion, sorry about that!
I tried to redo the pr but unfortunately, and for some weird reason, I can't do a pr on your own branch anymore.
So here is the fix commit I did, you can see the change. https://github.com/cuberite/cuberite/commit/f9d54a0ee707bbbb55b5f9e181a96e3e4a09f369 if you can just implement this commit, or if you can't just modify it by yourself ?
just by modifying this, the compilation check should pass!

Reply:
you still on it tho ?

Reply:
Don't worry. No stress at all. We're happy for anyone who wants to contribute! Work at your pace and possibilities